### PR TITLE
Fix typo in Readme

### PR DESCRIPTION
**Pull request by akleemans**
Gotta start somewhere ツ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,4 +36,4 @@ Running tests:
 - `make test`
 
 Running benchmarks:
-- `make bench`
\ No newline at end of file
+- `make bench`
```
Comment (akleemans) [inline README.md:39]: Not sure why this is showing up as changed.

**Review by Matt-Hurd — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Matt-Hurd) [reply README.md:39]: ¯\\\_(ツ)\_/¯

**Review by zdylag — APPROVED** [review]
Heck! Good find!

**Review by zdylag — CHANGES_REQUESTED** [review]
Actually, would you mind changing the commit message to `Readme: Fix typo in Readme`?

File: README.md (hunk 1 above)
Comment (typwo) [reply README.md:39]: Probably just ```text=auto``` doing its magic

**Comment by akleemans** [comment]
> Actually, would you mind changing the commit message to `Readme: Fix typo in Readme`?

Sure! Do you mean in a sense of `Readme: ...` being the "feature" that has been worked on in this PR?


Are there any contribution guidelines yet? (also regarding squash vs. rebase, commit message guidelines)
If not, we could just create them.

**Comment by zdylag** [comment]
I think I have been referencing gos commit format (https://golang.org/doc/contribute#commit_messages), but yes that also sounds like a good item to add to the issues list!

**Comment by akleemans** [comment]
@zdylag Changed the commit message, ready for review

**Review by zdylag — APPROVED** [review]
👍

**Pull request merged**